src/decisionengine/framework/modules/de_logger.py

```diff
@@ -4,9 +4,25 @@
 import os
 import logging
 import logging.handlers
+import logging.config
+import structlog
+import decisionengine.framework.modules.logging_configDict as configDict
 
 MB = 1000000
 
+logger = structlog.getLogger("decision_engine")
+logger = logger.bind(module=__name__.split(".")[-1])
+
+exception_logger = logging.getLogger("exception")
+exception_logger.setLevel(logging.ERROR)
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(module)s - %(threadName)s - %(levelname)s - %(message)s",
+                              datefmt="%Y-%m-%dT%H:%M:%S%z")
+fh = logging.FileHandler('/var/log/decisionengine/exception.log')
```

Comment:
This should probably be rigged up in a way that lets us swap out the name via the config (and unit tests).

Reply:
Yes, you are right. I will make this change.

Reply:
fixed. that exception logger is no longer needed.